### DISCUSSION TITLE
Communication with the API failed

### DISCUSSION THREAD

**Discussion by tomasenskede** [nginx-proxy/nginx-proxy#2210] [General]
Clean install Nginx in Docker.

When I try to ad an _Let's Encrypt_ SSL Cert and test the server reachability... I get;

**Communication with the API failed, is NPM running correctly?**

when I try to an SSL to an host; I got this error...  **Internal Error**

Whats wrong?

**Comment by buchdag** [comment]
> ## Bugs
> 
> For bugs, the easier you make it to reproduce the issue you see and the more initial information you provide, the easier and faster the bug can be identified and can get fixed.
> 
> Please at least provide:
> - the exact nginx-proxy version you're using (if using `latest` please make sure it is up to date and provide the version number printed at container startup).
> - complete configuration (compose file, command line, etc) of both your nginx-proxy container(s) and proxied containers. You should redact sensitive info if needed but please provide **full** configurations.
> - generated nginx configuration obtained with `docker exec nameofyournginxproxycontainer nginx -T`
> 
> If you can provide a script or docker-compose file that reproduces the problems, that is very helpful.
